src/components/reservation-button/reservation-button.component.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { selectSelectedHosting } from '../../redux/reservation/reservation.selectors';
+
+import './reservation-button.styles.scss';
+
+export function ReservationButton({ handleClick, size = 'sm' }) {
```

Comment:
- [ ] When I try to add a reservation, this happens:


![image](https://user-images.githubusercontent.com/57421823/203672125-4aaaf456-c342-48d7-ad37-741389f36bad.png)

Kindly fix this bug so your app works as intended 👍